src/frontend/magnify/package.json

```diff
@@ -75,11 +78,16 @@
     "grommet": "2.24.0",
     "polished": "4.2.2",
     "react": "18.2.0",
-    "react-intl": "6.0.4",
     "react-query": "3.39.1",
     "react-router-dom": "6.3.0",
     "styled-components": "5.3.5",
-    "validator": "13.7.0"
+    "validator": "13.7.0",
+    "formik": "2.2.9",
+    "yup": "0.32.11",
+    "grommet-icons": "4.8.0",
+    "react-intl": "6.0.4"
```

Comment:
Ditto

Please sort peerDependencies alphabetically to prevent futur modification when you will install new deps.